modules/ocf/files/puppet-trigger

```diff
@@ -32,8 +32,7 @@ def switch_to_environment(env):
         environment=env,
     )
 
-    # TODO: use subprocess.run on python3.5; we don't actually care about stdout here
-    subprocess.check_output(
+    subprocess.run(
```

Comment:
Jessie still has [Python 3.4](https://packages.debian.org/jessie/python3) and not 3.5, so I don't think this can be done until we fully upgrade to stretch.